### PR TITLE
Support run once via interval argument

### PR DESCRIPTION
**Pull request by xushiyan**
To allow run-once mode by setting polling interval to non-positive number.

### CODE REVIEW

**Comment by xushiyan** [comment]
Hi @zentol I'd like to add this feature. However, I'm not sure why 9b844dc64c74694d0c1ebab8834a4eb0b5e1bbd4 was introduced. It seems to create a double while-true-loop given `com.ververica.CiBot#run` already running in a loop and catches exceptions within itself.

File: src/main/java/com/ververica/CiBot.java [hunk 1]
```diff
@@ -142,7 +142,8 @@ public CiBot(Core core, GitActionsImpl gitActions, int pollingIntervalInSeconds,
 	public void run() {
 		try {
 			Date lastUpdateDate = Date.from(Instant.now().minus(Duration.ofHours(backlogHours)));
-			while (true) {
+			boolean runContinuously = pollingIntervalInSeconds > 0;
+			do {
```
Comment (zentol) [inline src/main/java/com/ververica/CiBot.java]: let's move the inner `try` block into a separate method, and structure `run` like this:

```
if (runContinuously) {
	while (true) {
		internalRun()
		<sleep>
	}
} else {
	internalRun()
}
```

**Review by zentol — CHANGES_REQUESTED** [review]
`com.ververica.CiBot#run` does not catch exceptions in the loop, because it is simpler to just restart the whole thing if an error occurred than trying to account for possible inconsistencies in the internal state. Until 9b844dc64c74694d0c1ebab8834a4eb0b5e1bbd4 this was a manual step, which became a bit annoying over time.

As for the propose change, I had a comment on the code structuring.

**Comment by xushiyan** [comment]
@zentol sorry this slipped through me...thanks a lot for getting this landed!

**Pull request merged**